test/inductor/test_compiled_optimizers.py

```diff
@@ -379,6 +380,49 @@ def fn():
         self.assertTrue(p_ref() is None)
         gc.enable()
 
+    @requires_cuda
```

Comment:
Why does this require CUDA? Could this run for both CPU and CUDA?

Reply:
Yup it does with the new commit